src/BitWriter_3_2/BitcodeWriter.cpp

```diff
@@ -0,0 +1,1919 @@
+//===--- Bitcode/Writer/BitcodeWriter.cpp - Bitcode Writer ----------------===//
```

Comment:
Is this file copied from another place entirely? Is that why it doesn't conform to Halide conventions?